composer.json

```diff
@@ -10,10 +10,10 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": "^7.4 || ^8.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
-        "slevomat/coding-standard": "^7.0",
-        "squizlabs/php_codesniffer": "^3.6",
-        "webimpress/coding-standard": "^1.2"
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+        "slevomat/coding-standard": "^8.3.0",
```

Comment:
@geerteltink this PR was about bumping `slevomat/coding-standard`: it would be OK to bump to any latest `^7.x`, but `^8.x` is a breaking change, and we can't have that in `2.x`.

Reply:
Thinking about it, I see what you mean. Someone can have their own custom rule set.

slevomat 7 is usable anyway. This PR is to bump to version 8. So I guess merging this into version 3.x is fine. There is no point in updating to 7.x.x. If you install the coding standard you will have the latest 7.x.x ruleset anyway.

Reply:
I think that before we move this to 3.x, we need to make sure (tomorrow, next TSC meeting) that everyone is on board with having the next release of `laminas/laminas-coding-standard` be a major one, perhaps with some long-awaited BC breaks in it.